### PR TITLE
Added CVE-2016-9593

### PR DESCRIPTION
**Pull request by lzap**

### CODE REVIEW

File: security.md [hunk 1]
```diff
@@ -67,6 +68,16 @@ All security advisories made for Foreman are listed below with their correspondi
 
 ### Disclosure details
 
+#### <a id="2016-9593"></a>CVE-2016-9593: foreman-debug collects sensitive data
+
+It was found that foreman-debug did not obfuscate sensitive information (such as passwords) from the katello-installer log file, allowing a user authorized to access the log files created by foreman-debug to gain access to potentially sensitive information.
```
Comment (domcleal) [inline security.md]: 1. foreman-installer
1. Also answer and other configuration files, not only log files.
1. "the log files created by foreman-debug" should be "archives created by foreman-debug".

File: security.md [hunk 2]
```diff
@@ -67,6 +68,16 @@ All security advisories made for Foreman are listed below with their correspondi
...
+
+This issue was reported by Pavel Moravec.
+
+* Affects all Foreman versions up to 1.14.x
```
Comment (domcleal) [inline security.md]: 1.2.0 up to and including 1.14.x

File: security.md [hunk 3]
```diff
@@ -67,6 +68,16 @@ All security advisories made for Foreman are listed below with their correspondi
...
+* Fix released in Foreman 1.15.0
```
Comment (domcleal) [inline security.md]: Fix is pending release

**Review by domcleal — CHANGES_REQUESTED** [review]

**Comment by lzap** [comment]
Done.

File: security.md [hunk 4]
```diff
@@ -67,6 +68,16 @@ All security advisories made for Foreman are listed below with their correspondi
...
 ### Disclosure details
 
+#### <a id="2016-9593"></a>CVE-2016-9593: foreman-debug collects sensitive data
+
+It was found that foreman-debug did not obfuscate sensitive information (such as passwords, answer and configuration options) from the foreman-installer log file, allowing a user authorized to access the archives created by foreman-debug to gain access to potentially sensitive information.
+
+This issue was reported by Pavel Moravec.
+
+* Affects Foreman 1.2.0 up to and including 1.14.x
+* Fix is pending release Foreman 1.15.0
```
Comment (domcleal) [inline security.md]: "pending release _in_ Foreman [..]", or just "pending release"

**Review by domcleal — CHANGES_REQUESTED** [review]

File: security.md (hunk 4 above)
Comment (lzap) [reply security.md]: Fixed.

**Comment by domcleal** [comment]
Merged, thanks.

**Pull request merged**